__tests__/components.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { configure, shallow, mount } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+
+import App from '../client/components/App';
+
+// Newer Enzyme versions require an adapter to a particular version of React
+configure({ adapter: new Adapter() });
+
+describe('React unit tests', () => {
+  let wrapper;
+
+  beforeAll(() => {
+    wrapper = shallow(<App />);
+  });
+
+  it('renders without crashing', () => {
+    const div = document.createElement('div');
```

Comment:
Hi Liam, this test failed for some reason.
Also, it doesn't have any expected condition to test against.
I'd suggest to insert something like this `expect(wrapper.find('header').text()).toEqual('<HeaderNav />');`. instead of lines 19-21.